### PR TITLE
tools/tidb-lightning: update TiDB Lightning FAQ and troubleshooting

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR:
* in `faq/tidb-lightning.md`, link to "Troubleshooting" which provides details on how to handle errors
* elaborates some FAQ items
* elaborates the meaning of labels of some metrics
* refines the language

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1535
### Which version does your change affect? 
dev, v3.0 and v3.1

### CODE REVIEW

**Comment by anotherrachel** [comment]
@kennytm @yikeke PTAL, thanks!

File: dev/faq/tidb-lightning.md [hunk 1]
```diff
@@ -34,6 +40,28 @@ TiDB Lightning 需要以下权限：
...
 
+## 如何正确重启 TiDB Lightning？
+
+根据 `tikv-importer` 的状态，重启 TiDB Lightning 的基本顺序如下：
+
+如果 `tikv-importer` 仍在运行：
+
+1. [结束 `tidb-lightning` 进程](#如何正确结束-tidb-lightning-进程)。
+2. 执行修改操作（如修复数据源、更改设置、更换硬件等）。
+3. 如果之前的修改更改了任何表，[清除对应的断点](/dev/reference/tools/tidb-lightning/checkpoints.md#--checkpoint-remove)。
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
3. 如果上面的修改操作更改了任何表，你还需要[清除对应的断点](/dev/reference/tools/tidb-lightning/checkpoints.md#--checkpoint-remove)。
```

File: dev/faq/tidb-lightning.md [hunk 2]
```diff
@@ -57,13 +85,13 @@ ADMIN CHECKSUM TABLE `schema`.`table`;
 
 ## TiDB Lightning 支持哪些格式的数据源？
 
-到 v2.1.6 版本为止，只支持本地文档形式的数据源，支持 [Mydumper](/dev/reference/tools/mydumper.md) 或 [CSV](/dev/reference/tools/tidb-lightning/csv.md) 格式。
+TiDB Lightning 只支持 [Mydumper](/dev/reference/tools/mydumper.md) 生成的 SQL dump 或本地 [CSV](/dev/reference/tools/tidb-lightning/csv.md) 格式的数据源。
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
TiDB Lightning 只支持两种格式的数据源：

1. [Mydumper](/dev/reference/tools/mydumper.md) 生成的 SQL dump
2. 储存在本地文件系统的 [CSV](/dev/reference/tools/tidb-lightning/csv.md) 文件
```

File: dev/faq/tidb-lightning.md [hunk 3]
```diff
@@ -118,7 +146,15 @@ tidb-lightning-ctl --switch-mode=normal
...
 ## TiDB Lightning 可以使用千兆网卡吗？
 
-使用 TiDB Lightning 必须配置万兆网卡。**不能使用**千兆网卡，尤其是在部署 `tikv-importer` 的机器上。千兆网卡的总带宽只有 120 MB/s，而且需要与整个 TiKV 集群共享。在使用 TiDB Lightning 导入时，极易用尽所有带宽，继而因 PD 无法联络集群使集群断连。
+使用 TiDB Lightning 建议配置万兆网卡。**不推荐**使用千兆网卡，尤其是在部署 `tikv-importer` 的机器上。
+
+千兆网卡的总带宽只有 120 MB/s，而且需要与整个 TiKV 集群共享。在使用 TiDB Lightning 导入时，极易用尽所有带宽，继而因 PD 无法联络集群使集群断连。为了避免这种情况，你可以在 [`tikv-importer` 的配置文件](/dev/reference/tools/tidb-lightning/config.md#tikv-importer-配置参数)中**限制上传速度**。
+
+```toml
+[import]
+# Importer 上传至 TiKV 的最大速度（字节/秒）。
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
# Importer 上传至 TiKV 的最大速度（字节/秒）。
# 建议将该速度设为 100 MB/s 或更小。
```

File: dev/faq/tidb-lightning.md [hunk 4]
```diff
@@ -129,4 +165,20 @@ tidb-lightning-ctl --switch-mode=normal
...
+
+    ```sh
+    tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-remove=all
+    ```
+
+    如果出于某些原因而无法运行该命令，你可以尝试手动删除 `/tmp/tidb_lightning_checkpoint.pb` 文件。
+
+2. 删除 `tikv-importer` 所在机器上的整个 “import” 文件目录。
+
+3. 如果需要的话，删除 TiDB 集群上创建的所有表和数据。
```
Comment (yikeke) [inline dev/faq/tidb-lightning.md]:
```suggestion
3. 如果需要的话，删除 TiDB 集群上创建的所有表和库。
```

File: dev/how-to/troubleshoot/tidb-lightning.md [hunk 5]
```diff
@@ -36,7 +36,8 @@ TiDB Lightning 的正常速度为每条线程每 2 分钟导入一个 256 MB 的
 3. 如果数据源是由机器生成而不是从 Mydumper 备份的，需确保数据符合表的限制，例如：
 
     * 自增 (AUTO_INCREMENT) 的列需要为正数，不能为 0。
-    * 单一键和主键 (UNIQUE and PRIMARY KEYs) 不能有重复的值。
+    * 唯一键和主键 (UNIQUE and PRIMARY KEYs) 不能有重复的值。
+4. 如果 TiDB Lightning 启动失败后没有正确重启，可能会因为数据不同步而出现校验和不一致。
```
Comment (yikeke) [inline dev/how-to/troubleshoot/tidb-lightning.md]:
```suggestion
4. 如果 TiDB Lightning 之前失败停机过，但没有正确重启，可能会因为数据不同步而出现校验和不一致。
```
看原文似乎不是“启动失败”？

File: dev/how-to/troubleshoot/tidb-lightning.md [hunk 6]
```diff
@@ -125,3 +128,7 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
         ```
 
 2. 导出数据时，必须加上 `--skip-tz-utc` 选项。
+
+3. 确保整个集群使用的是同一最新版本的 `tzdata` (2018i 或更高版本)。
+
+    在 CentOS 上运行 `yun info tzdata` 命令查看 `tzdata` 的版本及是否有更新。运行 `yum upgrade tzdata` 命令升级 `tzdata`。
```
Comment (yikeke) [inline dev/how-to/troubleshoot/tidb-lightning.md]:
```suggestion
    如果你使用的是 CentOS 机器，你可以运行 `yun info tzdata` 命令查看 `tzdata` 的版本及是否有更新。然后运行 `yum upgrade tzdata` 命令升级 `tzdata`。
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 7]
```diff
@@ -228,74 +240,110 @@ scrape_configs:
...
-    - **state**：`pending` / `written` / `closed` / `imported` / `altered_auto_inc` / `checksum` / `analyzed` / `completed`
-    - **result**：`success` / `failure`
+    - **state**：表的状态，表明当前应执行的操作
+        * `pending` — 等待处理
+        * `written` — 所有数据已编码和传输
+        * `closed` — 所有对应的引擎文件已关闭
+        * `imported` — 所有引擎文件已上传到目标集群
+        * `altered_auto_inc` — 自增 ID 已改
+        * `checksum` — 已计算校验和
+        * `analyzed` — 已进行分析
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
        * `analyzed` — 已进行统计信息分析
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 8]
```diff
@@ -228,74 +240,110 @@ scrape_configs:
...
+    - **result**：当前操作的执行结果
+        * `success` — 成功
+        * `failure` — 失败（未完成）
 
 - **`lightning_chunks`**（计数器）
 
     计算处理过的 Chunks 及其状态。标签：
 
-    - **state**：`estimated` / `pending` / `running` / `finished` / `failed`
+    - **state**: 单个 Chunk 的状态，表明当前所处的阶段
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
    - **state**: 单个 Chunk 的状态，表明该 Chunk 当前所处的阶段
```

File: dev/reference/tools/tidb-lightning/monitor.md [hunk 9]
```diff
@@ -228,74 +240,110 @@ scrape_configs:
...
+        * `estimated` — （非状态）当前任务中 chunk 的数量
```
Comment (yikeke) [inline dev/reference/tools/tidb-lightning/monitor.md]:
```suggestion
        * `estimated` — （非状态）当前任务中 Chunk 的数量
```

**Comment by anotherrachel** [comment]
@kennytm PTAL, thanks!

File: dev/how-to/troubleshoot/tidb-lightning.md [hunk 10]
```diff
@@ -125,3 +128,7 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
         ```
 
 2. 导出数据时，必须加上 `--skip-tz-utc` 选项。
+
+3. 确保整个集群使用的是同一最新版本的 `tzdata` (2018i 或更高版本)。
+
+    如果你使用的是 CentOS 机器，你可以运行 `yun info tzdata` 命令查看 `tzdata` 的版本及是否有更新。然后运行 `yum upgrade tzdata` 命令升级 `tzdata`。
```
Comment (kennytm) [inline dev/how-to/troubleshoot/tidb-lightning.md]:
```suggestion
    如果你使用的是 CentOS 机器，你可以运行 `yum info tzdata` 命令查看 `tzdata` 的版本及是否有更新。然后运行 `yum upgrade tzdata` 命令升级 `tzdata`。
```

**Comment by anotherrachel** [comment]
v3.0 and v3.1 updated. PTAL, thanks! @kennytm @yikeke

File: v3.1/faq/tidb-lightning.md [hunk 11]
```diff
@@ -34,6 +40,28 @@ TiDB Lightning 需要以下权限：
...
+
+如果 `tikv-importer` 需要重启：
+
+1. [结束 `tidb-lightning` 进程](#如何正确结束-tidb-lightning-进程)。
+2. [结束 `tikv-importer` 进程](#如何正确结束-tikv-importer-进程)。
+3. 执行修改操作（如修复数据源、更改设置、更换硬件等）。
+4. 重启 `tikv-importer`。
+5. 重启 `tidb-lightning` 并等待，**直到程序因校验和错误（如果有的话）而失败**。
+    * 重启 `tikv-importer` 将清除所有仍在写入的引擎文件，但是 `tidb-lightning` 并不会感知到该操作。从 v3.0 开始，最简单的方法是让 `tidb-lightning` 继续，然后再重试。
+6. [清除失败的表及断点](/v3.1/how-to/troubleshoot/tidb-lightning.md#Checkpoint-for-has-invalid-status-错误码)。
```
Comment (yikeke) [inline v3.1/faq/tidb-lightning.md]: ditto

File: v3.0/faq/tidb-lightning.md [hunk 12]
```diff
@@ -35,6 +41,28 @@ TiDB Lightning 需要以下权限：
...
+
+如果 `tikv-importer` 需要重启：
+
+1. [结束 `tidb-lightning` 进程](#如何正确结束-tidb-lightning-进程)。
+2. [结束 `tikv-importer` 进程](#如何正确结束-tikv-importer-进程)。
+3. 执行修改操作（如修复数据源、更改设置、更换硬件等）。
+4. 重启 `tikv-importer`。
+5. 重启 `tidb-lightning` 并等待，**直到程序因校验和错误（如果有的话）而失败**。
+    * 重启 `tikv-importer` 将清除所有仍在写入的引擎文件，但是 `tidb-lightning` 并不会感知到该操作。从 v3.0 开始，最简单的方法是让 `tidb-lightning` 继续，然后再重试。
+6. [清除失败的表及断点](/v3.0/how-to/troubleshoot/tidb-lightning.md#Checkpoint-for-has-invalid-status-错误码)。
```
[the remark this answers is not in the document]
Comment (anotherrachel) [reply v3.0/faq/tidb-lightning.md]: Fixed in [855f5a0](https://github.com/pingcap/docs-cn/pull/2176/commits/855f5a0e0ab9130961bdf19d6d963bb824b2f93c) @yikeke

**Pull request merged**